src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
move it after class Animal

Reply:
fix it 
https://github.com/mate-academy/js_herbivores_and_carnivores/pull/1547/files#r1126014733

Reply:
You mean outside the class?
If yes, can you tell me why? Please.
Thanks